visualization/app/codeCharta/ui/searchBar/searchBar.component.ts

```diff
@@ -50,6 +50,10 @@ export class SearchBarController implements BlacklistSubscriber, SearchPatternSu
 		this.applyDebouncedSearchPattern()
 	}
 
+	clearSearchPattern() {
+		this.resetSearchPattern()
+	}
+
```

Comment:
Minor note: The new method is a bit verbose. We could change the accessability of `resetSearchPattern()` from private to public :)